reference-implementation/lib/helpers.js

```diff
@@ -83,15 +83,20 @@ exports.InvokeOrNoop = (O, P, args) => {
   return Call(method, O, args);
 };
 
-exports.PromiseInvokeOrNoop = (O, P, args) => {
-  assert(O !== undefined);
-  assert(IsPropertyKey(P));
-  assert(Array.isArray(args));
+exports.PromiseInvoke = (F, V, args) => {
```

Comment:
This is no longer an "Invoke", but instead a "Call". "Invokes" take (object, property name) pairs.

Reply:
Thanks, fixed.